docs/DevGuide/Protocols.md

```diff
@@ -22,7 +26,8 @@ protocol through metaprogramming or documentation, and unit tests that enforce
 conformance to these protocols.
```

Comment:
I think it might be good to emphasize already after the first sentence in the paragraph that protocols is not really like concepts and is trying to solve a different problem, but that the way protocols are implemented, they can easily be used with concepts. I think it would be good to have a short section comparing what concepts solves to what protocols solves, and how they complement each other.